User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+| **Posts**                                 | No more than one per hour, capped at three per day                                   | Limit per day is the greater of five or the number of previous well-received questions and answers.  | Full access                                      | Full access                                                                                         | Full access                                                                       | Full access                                                    |
+| **Responses**                               | No more than one per fifteen minutes                                                 | No change                                                                                            | Full access                                      | Full access                                                                                         | Full access                                                                       | Full access                                                    |
+| **Comments**                              | Limited to his own posts and responses                                                 | May post up to ten per day on others' posts and responses                                          | Full access                                      | Full access                                                                                         | Full access                                                                       | Full access                                                    |
+| **Flags**                                 | Limited to responses and comments on his own posts, capped at three pending in two days | Full access                                                                                          | Full access                                      | Full access                                                                                         | Full access                                                                       | Full access                                                    |
+| **Upvotes**                               | Limited to responses on his own posts                                                  | Full access                                                                                          |                                                  | Full access                                                                                         | Full access                                                                       | Full access                                                    |
```

Comment:
We should try to avoid singular personal pronouns.  Consider speaking to the user, i.e. second person ("you", "your").